test/types.ts

```diff
@@ -50,32 +41,32 @@ describe('Types', () => {
   describe('ES6 exports', () => {
 
     it('HttpStatus', () => {
-      const statuses: HttpStatus = status
```

Comment:
Why are you removing the type cast to HttpStatus ? I believe we shall keep those (same question for the tests below)

Reply:
Because they are now implicitly inferred.

Do you wish them to stay?

Reply:
I just reworked it. It now only has necessary changes.